generate_demog_maps.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import json
+from core_data_modules.cleaners import Codes
+import sys
+
+from core_data_modules.logging import Logger
+from core_data_modules.traced_data.io import TracedDataJsonIO
+
+from src.lib import PipelineConfiguration
+
+Logger.set_project_name("IMAQAL")
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generates graphs for analysis")
+
+    parser.add_argument("user", help="User launching this program")
+    parser.add_argument("messages_json_input_path", metavar="messages-json-input-path",
+                        help="Path to a directory with JSON files to read the TracedData of the messages data from")
```

Comment:
Describe the required format of the file names in this directory

Reply:
Updated thanks.

Reply:
Suspect you need to push